azure/requests.yaml

```diff
@@ -27,3 +27,7 @@ releases:
   requests:
   - name: cert-exporter
     version: ">= 2.9.2"
+- name: "> 25.0.1 < 26.0.0 || > 26.0.1 < 27.0.0 || > 27.0.1 < 28.0.0 || > 28.0.1 < 29.0.0"
```

Comment:
This would first affect v25.0.2, v26.0.2 and so on. Also, what about v29 releases? Don't they need Cilium app v0.25.2?